install/install.sh

```diff
@@ -41,6 +41,9 @@ do
                     exit 1
                 fi
             fi
+	    if [  -f "$file" ]; then
```

Comment:
This if is still not aligned correctly :(

Reply:
I tried to change it on editor as well as implement change on git hub directly but it isn’t just working :|